examples/notebooks/openEO_Python_Sentinel2_EURAC.ipynb

```diff
@@ -126,7 +126,7 @@
     "from matplotlib.colors import NoNorm\n",
     "from matplotlib_scalebar.scalebar import ScaleBar\n",
     "import numpy as np\n",
-    "from osgeo import gdal\n",
+    "import gdal\n",
```

Comment:
as far as I understand `import gdal` is deprecated and `from osgeo import gdal` should be used instead